From 3af148bf0eb1bdf01885fb69198ab8d5741e597f Mon Sep 17 00:00:00 2001
From: Harman Singh <harman.singh@student.howest.be>
Date: Sat, 25 Nov 2023 19:44:58 +0100
Subject: [PATCH 06/13] update text so it matches our company

---
 src/components/settings/PrivacyAndPersonalization.jsx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/components/settings/PrivacyAndPersonalization.jsx b/src/components/settings/PrivacyAndPersonalization.jsx
index 764efe5..8c73763 100644
--- a/src/components/settings/PrivacyAndPersonalization.jsx
+++ b/src/components/settings/PrivacyAndPersonalization.jsx
@@ -67,7 +67,7 @@ const PrivacyAndPersonalization = () => {
 								Marketing Communication
 							</h3>
 							<p className="font-light">
-								Learn about new courses and programs,
+								Learn about new exploration areas,
 								scholarship opportunities and upcoming special
 								events.
 							</p>
-- 
2.37.0.windows.1

