From d149305802e45d4a56215a9874c8717be97631b3 Mon Sep 17 00:00:00 2001
From: Harman Singh <harman.singh@student.howest.be>
Date: Sat, 25 Nov 2023 19:47:47 +0100
Subject: [PATCH 11/13] update prop validation, wrong type

---
 .../subscriptions-overview/SubscriptionsOverviewCard.jsx        | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/components/subscriptions-overview/SubscriptionsOverviewCard.jsx b/src/components/subscriptions-overview/SubscriptionsOverviewCard.jsx
index f504002..e25327c 100644
--- a/src/components/subscriptions-overview/SubscriptionsOverviewCard.jsx
+++ b/src/components/subscriptions-overview/SubscriptionsOverviewCard.jsx
@@ -7,7 +7,7 @@ const SubscriptionsOverviewCard = ({ name, description, price, period, features
 	SubscriptionsOverviewCard.propTypes = {
 		name: PropTypes.string.isRequired,
 		description: PropTypes.string.isRequired,
-		price: PropTypes.number.isRequired,
+		price: PropTypes.string.isRequired,
     period: PropTypes.string.isRequired,
 		features: PropTypes.array.isRequired,
 	};
-- 
2.37.0.windows.1

