From 32ecd0ac54f50dab6f01b9e02f26a9d2c2f3c9a8 Mon Sep 17 00:00:00 2001
From: Harman Singh <harman.singh@student.howest.be>
Date: Sat, 25 Nov 2023 19:50:03 +0100
Subject: [PATCH 13/13] add width to both sections + decrease shadow

---
 src/components/transactions/Transactions.jsx | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/components/transactions/Transactions.jsx b/src/components/transactions/Transactions.jsx
index e1a1e8e..2156fd9 100644
--- a/src/components/transactions/Transactions.jsx
+++ b/src/components/transactions/Transactions.jsx
@@ -59,7 +59,7 @@ const Transactions = () => {
     return (
         <section>
             <Title content="Transactions" />
-            <div className="bg-white w-full px-8 pb-6 mb-12 max-h-[335px] overflow-y-auto scrollbar scrollbar-thumb-brandCyan scrollbar-track-transparent shadow">
+            <div className="bg-white w-8/12 px-8 pb-6 mb-12 max-h-[335px] overflow-y-auto scrollbar scrollbar-thumb-brandCyan scrollbar-track-transparent shadow-md">
                 <table>
                     <thead className="sticky top-0 bg-white w-full">
                         <tr className="text-left">
@@ -102,7 +102,7 @@ const Transactions = () => {
             </div>
 
             <Title content="Auction Proceeds" />
-            <div className="bg-white w-full px-8 pb-6 max-h-[335px] overflow-y-auto scrollbar scrollbar-thumb-brandCyan scrollbar-track-transparent shadow">
+            <div className="bg-white w-8/12 px-8 pb-6 max-h-[335px] overflow-y-auto scrollbar scrollbar-thumb-brandCyan scrollbar-track-transparent shadow-md">
                 <table>
                     <thead className="sticky top-0 bg-white w-full">
                         <tr className="text-left">
-- 
2.37.0.windows.1

